container/docker/factory.go

```diff
@@ -172,9 +172,12 @@ func (self *dockerFactory) DebugInfo() map[string][]string {
 	return map[string][]string{}
 }
 
+var (
+	version_regexp_string = "(\\d+)\\.(\\d+)\\.(\\d+)"
+	version_re            = regexp.MustCompile(version_regexp_string)
```

Comment:
Ah, makes sense. Maybe move the declaration to just above the function in that case? Also, consider making the string const.